psst-gui/src/ui/artist.rs

```diff
@@ -82,18 +78,33 @@ fn async_related_widget() -> impl Widget<AppState> {
         )
 }
 
-pub fn artist_widget() -> impl Widget<Artist> {
-    let artist_image = cover_widget(theme::grid(7.0));
-    let artist_label = Label::raw()
-        .with_font(theme::UI_FONT_MEDIUM)
-        .lens(Artist::name);
-    let artist = Flex::row()
+pub fn artist_widget(horizontal: bool) -> impl Widget<Artist> {
+    let mut artist = if horizontal { Flex::column() } else { Flex::row() };
+
+    let artist_image = cover_widget(if horizontal { theme::grid(16.0) } else { theme::grid(6.0) });
+    artist = artist
         .with_child(artist_image)
-        .with_default_spacer()
-        .with_flex_child(artist_label, 1.0);
-    artist
-        .padding(theme::grid(0.5))
+        .with_default_spacer();
+
+    artist = if horizontal {
+        artist.with_child(Label::raw()
+            .with_font(theme::UI_FONT_MEDIUM)
+            .lens(Artist::name))
+    } else {
+        artist.with_flex_child(Label::raw()
+            .with_font(theme::UI_FONT_MEDIUM)
+            .lens(Artist::name), 1.0)
+    };
+    
+    let artist = if horizontal {
+        artist.fix_width(theme::grid(16.0)).padding_horizontal(theme::grid(1.0)).align_left()
+    } else {
+        artist.align_left()
```

Comment:
Rather than breaking them up into if else for every method applied, I think it would be more readable to put them in a single if/else horizontal statement.